### ISSUE TITLE
Make configurations in remote and parameter nodes update values in linked controls

### ISSUE THREAD

**Issue by rubberduck9999** [RodZill4/material-maker#693]
**Feature/enhancement description:**
In this simple example I made a material with presets to choose from, let's say you have a preset "blue" to set the color to blue, which it does in the node itself, that is also driven by a linked control, so the "Material Color" setting should change to blue too.

This way I can define presets, but also use the other input fields to modify the values without the need to check, which values changed to what.

It could be even as a separate option (next to the 3 existing, linked control, configurations and named parameter) and could show a "*" when some values are changed next to the preset option
![MM_presets](https://github.com/RodZill4/material-maker/assets/82282845/e7f41d98-6fb1-4bb9-a0d7-e73edd132eea)

**Comment by RodZill4** [comment]
I don't understand the difference between what you suggest and configuration parameters.

**Comment by rubberduck9999** [comment]
yeah, the difference isn't really that much, the main thing is, that if I set it to a different preset / configuration, the values in the input field get updated too.
I made previously my own custom scratch generator (where I even extended the splatter node to have random x and y scale values). I have both parameters to set, but also setup some presets.


And the Idea I had there, is to show an "*" like in this image, if you changed some default values from linked parameters, though I think this is optional
![grafik](https://github.com/RodZill4/material-maker/assets/82282845/01369a11-ab7a-40b0-8ce2-f89d13d5cd19)
In the case I change the preset, the values in the input fields are still the same, which is confusing when editing them, as it changes to something else, not representing the value of the preset which I may want to modify more or less
![grafik](https://github.com/RodZill4/material-maker/assets/82282845/3bcbe08e-d0e2-4052-ad7c-9a76f385f75d)

Though I think such presets could be even a separate feature, so that a user can quickly create own presets



[scratches_v2.zip](https://github.com/RodZill4/material-maker/files/15370545/scratches_v2.zip)
